### DISCUSSION TITLE
Stuck on Current state: LEAVING_POOL after leaving flexpool.io

### DISCUSSION THREAD

**Discussion by valenciakarlos** [Chia-Network/chia-blockchain#16827] [Support]
Seems like flexpool.io closed (i didnt get any notification but that's for another day). I tried "chia plot nft leave -i <id>" and created a self pool with "chia plotnft create -s"   the thing is that when i do "chia plotnft show" all my plots appear under the old pool and not the one I just created. See below, what can I do to resolve this situation and continue my farming?

carlos@chia730:~$ chia plotnft show
Wallet height: 4515658
Sync status: Synced
Wallet ID: 4
Current state: LEAVING_POOL
Current state from block height: 4515446
Launcher ID:  <removed jic>
Target address (not for plotting):  <removed jic>
Number of plots: 516
Owner public key:  <removed jic>
Pool contract address (use ONLY for plotting - do not send money to this address):  <removed jic>
Target state: SELF_POOLING
Target pool URL: None
Expected to leave after block height: 4515546

Wallet ID: 7
Current state: SELF_POOLING
Current state from block height: 4515474
Launcher ID: <removed jic>
Target address (not for plotting):  <removed jic>
Number of plots: 0
Owner public key:  <removed jic>
Pool contract address (use ONLY for plotting - do not send money to this address):  <removed jic>
Claimable balance: 0.0 xch (0 mojo)

**Comment by BrandtH22** [comment]
Hey @valenciakarlos , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

As for the pools, when you ran the command `chia plotnft create -s` you had actually created a second plotNFT.
Note that all of your plots are tied to a plot NFT and it is the plotNFT that needs to be migrated from one pool to another rather than the plots themselves.

This all said, it shows that your leave command will have completed at block height 4515546 so if you run the chia wallet show command again now since that height has lapsed you should see it in a self-pooling state now.

The extra plotNFT will unfortunately have to remain in the wallet as there is currently no way to remove or burn those records.

Does this clear up some of the confusion? Do you have any additional questions?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
